jobs/pacman-data-shipper/src/main/java/com/tmobile/cso/pacman/datashipper/Main.java

```diff
@@ -86,12 +75,17 @@ public static void main(String[] args) {
      * Ship data.
      * This method will be executed as When Shipper jar is getting executed as part of
      * Batch job(i.e) data-shipper-gcp-job,data-shipper-azure-job
+     *
      * @param params the params
      * @return
      */
     public static Map<String, Object> shipData(Map<String, String> params) {
+        LOGGER.info("Inside shipData Method");
+        LOGGER.debug("Shipper Job Params:{}", params);
         String jobName = System.getProperty("jobName");
         List<Map<String, String>> errorList = new ArrayList<>();
+        //this map contains all datasource values which should look for dynamic src location for Shipment
+        Set<String> datasourceForOverrideSourcePath = new HashSet<>(Arrays.asList("gcp", "redhat"));
```

Comment:
Why do we need this? The goal of generic is to not have these data sources hard coded.

Shouldn't the logic simply by:
If override folder field exists and not empty use that path, otherwise use s3.data